src/backersManager/rewardDistributionFinished.ts

```diff
@@ -0,0 +1,21 @@
+import { RewardDistributionFinished as RewardDistributionFinishedEvent } from "../../generated/BackersManagerRootstockCollective/BackersManagerRootstockCollective";
+import { Cycle } from "../../generated/schema";
+import { BigInt } from "@graphprotocol/graph-ts";
+
+export function handleRewardDistributionFinished(
+  event: RewardDistributionFinishedEvent
+): void {
+  let cycle = Cycle.load(event.address);
+  if (cycle == null) {
```

Comment:
Can this happen? that a distribution has finished and the cycle doesn't exist?

Reply:
updated